common/collision.go

```diff
@@ -139,6 +148,12 @@ type collisionEntity struct {
 	*SpaceComponent
 }
 
+type Collisioner interface {
```

Comment:
You switched naming conventions, this can be confusing. You went from `SystemInterface` to `Systermer`